mmrotate/core/bbox/coder/__init__.py

```diff
@@ -1,10 +1,11 @@
 # Copyright (c) OpenMMLab. All rights reserved.
+from .angle_coder import CSLCoder
```

Comment:
The name of `angle_coder` is not clear. We'd better rename it to `csl_coder` or another name.

Reply:
`angle_coder` will continue to add the `DCLCoder`, so it is not suitable to be renamed to `csl_coder`. Actually, I think the word angle has made the meaning quite clear.